ext/libxml/libxml.c

```diff
@@ -752,7 +752,9 @@ PHP_LIBXML_API void php_libxml_initialize(void)
 {
 	if (!_php_libxml_initialized) {
 		/* we should be the only one's to ever init!! */
+		ZEND_IGNORE_LEAKS_BEGIN();
 		xmlInitParser();
+		ZEND_IGNORE_LEAKS_END();
```

Comment:
So, only one remaining case ... it would of course be great if we could avoid the suppression. In particular I wonder whether it would be possible to call `xmlDestroyParser()` from `zend_post_shutdown_cb()`, which runs very late (in particular also after all mshutdown functions, which might still refer to libxml memory).

Unfortunately the original commit removing the call was not very clear on what exactly the issue being fixed is.

Reply:
If you don't have time to look into this though, feel free to commit this PR as is.

Reply:
I have not been able to get further information regarding that fix or the underlying issue. It seems to me that moving the call to `xmlDestroyParser()` to `zend_post_shutdown_cb()` is worth a try for master, but for now ignoring just these leaks looks acceptable to me.